src/libraries/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -2303,6 +2303,35 @@ private static BigInteger Subtract(uint[]? leftBits, int leftSign, uint[]? right
             return !right.Equals(left);
         }
 
+        /// <summary>
+        /// Gets the number of bits required to represent this BigInteger instance in two's complement notation without the sign bit.
+        /// </summary>
+        /// <returns>The non-negative number of bits in two's complement notation without the sign bit.</returns>
```

Comment:
In two's complement notation, -2 is …1110<sub>2</sub>; all the ones are sign bits, so I'd expect the method to return 1. From the tests however, it appears that `x.GetBitLength() == (-x).GetBitLength()` always. I am not sure that "two's complement notation" correctly describes the behavior of this method. If the documentation instead talked about the number of bits in the absolute value, then it would not need to say anything about sign bits.

Reply:
Thanks! The two's complement representation is actually the intended detail, missing from the implementation (and quite honestly, from the API review).